spyder/app/tour.py

```diff
@@ -217,19 +223,23 @@ def get_tour(index):
               'interact': True},
 
              {'title': _("Files"),
-              'content': _("This pane lets you navigate through the directories "
-                           "and files present in your computer.<br><br>"
-                           "You can also open any of these files with its "
-                           "corresponding application, by doing a double "
-                           "click on it.<br><br>"
-                           "There is one exception to this rule: plain-text "
-                           "files will always be opened in the Spyder Editor."),
+              'content': _("This pane lets you browse the files and "
+                           "directories on your computer.<br><br>"
+                           "You can open any file in its "
+                           "corresponding application by double-clicking it, "
+                           "and supported file types will be opened right "
+                           "inside of Spyder.<br><br>"
+                           "The Files pane also allows you to copy one or "
+                           "many absolute or relative paths, automatically "
+                           "formatted as Python string or list, and perform "

```

Comment:
```suggestion
                           "formatted as Python strings or list, and perform "
```

Not sure if that's a small grammatical error.

Reply:
It is, but list also needs to be plural to match.

```suggestion
                           "formatted as Python strings or lists, and perform "

```